Public/Src/Cache/ContentStore/Distributed/NuCache/CheckpointManager.cs

```diff
@@ -128,37 +131,16 @@ private async Task CreateFullCheckpointAsync(OperationContext context, EventSequ
             await _checkpointRegistry.RegisterCheckpointAsync(context, checkpointId, sequencePoint).ThrowIfFailure();
         }
 
-        private async Task<bool> CreateCheckpointIncrementalAsync(OperationContext context, EventSequencePoint sequencePoint, bool successfullyUpdatedIncrementalState)
+        private async Task CreateCheckpointIncrementalAsync(OperationContext context, EventSequencePoint sequencePoint)
         {
             InitializeIncrementalCheckpointIfNeeded(restoring: false);
 
-            BoolResult result = BoolResult.Success;
             var incrementalCheckpointsPrefix = $"incrementalCheckpoints/{sequencePoint.SequenceNumber}.{Guid.NewGuid()}.";
-            var newCheckpointInfo = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+            var newCheckpointInfo = new ConcurrentDictionary<string, string>(StringComparer.OrdinalIgnoreCase);

```

Comment:
Comment describing meaning of keys and values.

Reply:
Added a comment to see the comment of _incrementalCheckpointInfo field.

---
In reply to: [319296827](https://github.com/microsoft/BuildXL/pull/818#discussion_r319296827) [](ancestors = 319296827)